### DISCUSSION TITLE
📢 Exciting News: Introducing "Blueprint Display Name" !

### DISCUSSION THREAD

**Discussion by shirabendor-quali** [QualiTorque/qualitorque.github.io#318] [Announcements]
We are happy to announce the introduction of a highly anticipated feature called "Blueprint Display Name." 
This new capability empowers blueprint designers by enabling them to display blueprints with names that differ from their file names. Why is this important? Well, we understand that blueprint designers often adhere to guidelines and policies when it comes to Git file names. Unfortunately, these guidelines can sometimes lead to complicated names that are not user-friendly for our end users.

With the "Blueprint Display Name" feature, we aim to bridge this gap and provide a solution that ensures optimal usability for everyone. Now, blueprint designers can assign more user-friendly names to blueprints, enhancing the overall experience for all stakeholders involved.

Excited to give it a try? Simply head over to the catalog designer screen to explore and experience the power of this new capability firsthand.

**Comment by TomerAdmon** [comment]
Super cool! 🎉🎉
